lms/templates/navigation.html

```diff
@@ -78,6 +79,22 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
     </ol>
     <ol class="user">
       <li class="primary">
+          <form action="/i18n/setlang/" method="post" class="settings-language-form" id="settings-form">
+            <input type="hidden" id="csrf_token" name="csrfmiddlewaretoken" value="${csrf_token}">
+            <input title="preference api" aria-label="language" type="hidden" id="preference-api-url" value="${reverse('preferences_api', kwargs={'username': user.username})}">
+            <label for="settings-language-value"><span class="sr">${_("Choose Language")}</span></label>
```

Comment:
You don't have to worry about the ```for``` attribute or ```id``` attribute on the ```select``` if you use an implied ```label``` (put the closing ```</label>``` element after the closing```</select>``` .  It's not critical, but its less code and less likely to suffer a regression if code changes in the future.